src/ralph/ssl_certificates/models.py

```diff
@@ -27,10 +27,22 @@ class SSLCertificate(AdminAbsoluteUrlMixin, BaseObject):
         help_text=_('Full certificate name'),
         max_length=255
     )
+    domain_ssl = models.CharField(
```

Comment:
couldn't one certificate be used with multiple domains (ex. wildcard cert or cert with SAN)?

Reply:
Yes, it could be used. But every certificate have a common name and this is a main name which we are using to find a person who is responsible for it.